src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -496,7 +496,8 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundTryCastOperation(boundTryCast As BoundTryCast) As IConversionExpression
             Dim operand As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundTryCast.Operand))
             Dim syntax As SyntaxNode = boundTryCast.Syntax
-            Dim conversion As Conversion = _semanticModel.GetConversion(syntax)
+            Dim summary As BoundNodeSummary = New BoundNodeSummary(boundTryCast.Operand, boundTryCast, Nothing)
+            Dim conversion As Conversion = _semanticModel.GetTypeInfoForNode(summary).ImplicitConversion
```

Comment:
>Dim conversion As Conversion = _semanticModel.GetTypeInfoForNode(summary).ImplicitConversion [](start = 12, length = 92)

This feels wrong, all information is in bound tree, we should not be using SemanticModel to calculate it. Especially based on syntax.